modules/vstudio/vs2005_dotnetbase.lua

```diff
@@ -328,29 +328,27 @@
 -- Write out the references item group.
 --
 
-	dotnetbase.elements.references = function(prj)
+	dotnetbase.elements.references = function(prj, cfg)
```

Comment:
If we're putting a configuration condition on this `ItemGroup`, then I think everything in it should target that configuration. I would…

1. Change this line to pass only `cfg`, not `prj`
2. Update your changes to `assemblyReferences` to accept `cfg` as the only argument
3. Update `nuGetRefefences` to accept only `cfg`, and replace this line `local cfg = p.project.getfirstconfig(prj)` with `local prj = cfg.project`

I did a quick test and I believe these are all the changes that are needed. That should bring this `ItemGroup` into alignment with the rest of the code. Does that make sense? Let me know if you have questions.